controllers/utils.go

```diff
@@ -0,0 +1,13 @@
+package controllers
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func respondWithJson(w http.ResponseWriter, status int, payload interface{}) {
```

Comment:
Cool idea but it looks like we will need to handle this error. What if return some garbage from the database and the marshaler fails? It's not uncommon.